application/view/omeka/admin/media/show.phtml

```diff
@@ -46,6 +46,12 @@ $sectionNavs = [
         <div class="value"><?php echo $escape($media->mediaType()); ?></div>
     </div>
     <?php endif; ?>
+    <?php if (is_numeric($resource->size())): ?>
+    <div class="meta-group">
+        <h4><?php echo $this->translate('Size'); ?></h4>
+        <div class="value"><?php echo sprintf('%s bytes', number_format($resource->size())); ?></div>
```

Comment:
Possible issue with the use of `number_format` here... we probably should be using the locale-aware numberFormat, but we'd have to add that to our i18n helper to allow it work for people without intl... maybe not worth worrying about too much.

I don't think it should hold up merging but something to look at, possibly file an issue for.